apps/ideas/models/abstracts/impact_section.py

```diff
@@ -0,0 +1,93 @@
+from django.db import models
+
+CHALLENGE_TITLE = 'What problem or societal need are you working on?'
+CHALLENGE_HELP = ('Please look here for more information about the '
+                  'annual theme: (link to subpage with more '
+                  'detailed explanation of the annual theme). '
+                  '(max. 300 characters)')
+OUTCOME_TITLE = 'What would be a successful outcome for your project?'
+OUTCOME_HELP = ('If your project is selected, what will be different '
+                'a year from now? What will have changed? Please look '
+                'here for more information: (link to subpage '
+                'with explanation on impact). '
+                '(max. 300 characters)')
+PLAN_TITLE = ('How do you plan to get there? Remember to highlight'
+              ' what makes your project design or idea '
+              'different and innovative.')
+PLAN_HELP = ('Describe as concretely as possible the '
+             'approach and / or the method '
+             'you will use when implementing your idea. '
+             'What are the steps you plan '
+             'to take? Please look here for more information: '
+             '(link to subpage on with explanation on implementation). '
+             '(max. 500 characters)')
+IMPORTANCE_TITLE = 'Why is this idea important to you?'
+IMPORTANCE_HELP = ('What motivates you to bring this idea to life? '
+                   'What is your story? What is your personal '
+                   'connection and individual mission '
+                   'behind the idea? (max. 300 characters)')
+TARGET_GROUP_TITLE = 'Who are you doing it for?'
+TARGET_GROUP_HELP = ('Which target groups, stakeholders, '
+                     'beneficiaries or audiences are at the '
+                     'centre of your project? (max. 300 characters)')
+MEMBERS_TITLE = 'Who is in your project team?'
+MEMBERS_HELP = ('Please introduce us to the main members'
+                ' of your project team and briefly '
+                'summariese their experience and skills. '
+                '(max. 500 characters)')
+REACH_OUT_TITLE = ('Reach out – get feedback, '
+                   'ideas and inspiration from '
+                   'the Advocate Europe Community!')
+REACH_OUT_HELP = ('What kind of advice, comments or '
+                  'feedback would you like to receive '
+                  'about your idea from others on the '
+                  'platform? (max. 300 characters)')
+HOW_DID_YOU_HEAR_TITLE = ('How did you hear about '
+                          'Advocate Europe?')
+HOW_DID_YOU_HEAR_CHOICES = (
+    ('1', 'Personal contact'),
```

Comment:
Do we want to have the constants numeric? Since we are anyway storing them in charfields we can as well use strings, that are a bit more verbose. To make them easy to work with (in code), I would define them as Python enums.